common/locales/en/contrib.json

```diff
@@ -54,8 +54,9 @@
     "playerTiers": "Player Tiers",
     "tier": "Tier",
     "visitHeroes": "Visit the Hall of Heroes (contributors and backers)",
-    "conLearn": "Learn more about contributor rewards",
+    "conLearn": "<a href='http://habitica.wikia.com/wiki/Contributor_Rewards' target='_blank'>Learn more about contributor rewards</a>",
```

Comment:
@MathWhiz Here's an example of what I mean.  This `conLearn` string is actually a link, whereas the `conLearnLink` string below is just a URL, not an actual link.

Again, it's just a suggestion to clarify the intended usage.